src/main/kotlin/com/exactpro/th2/http/client/util/MessageUtil.kt

```diff
@@ -67,7 +67,7 @@ private const val HEADER_VALUE_SEPARATOR = ";"
 
 private fun createRequest(head: Message, body: RawMessage): RawHttpRequest {
     val metadata = body.metadata.propertiesMap
-    val method = head.getString(METHOD_FIELD) ?: metadata[METHOD_PROPERTY] ?: DEFAULT_METHOD
+    val method = (head.getString(METHOD_FIELD) ?: metadata[METHOD_PROPERTY])?.uppercase() ?: DEFAULT_METHOD
```

Comment:
Why do we need this?

Reply:
if incoming messages have lowercase method in request, no request are sent by client

Reply:
> no request are sent by client

Could you explain why?